sdk/keyvault/Azure.Security.Keyvault.Keys/src/KeyClient.cs

```diff
@@ -0,0 +1,175 @@
+﻿using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.Core.Pipeline.Policies;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Azure.Security.Keyvault.Keys
+{
+    public class KeyClient
```

Comment:
Need to add a few missing APIs:
  - GetAll / GetAllAsync
  - GetAllVerions / GetAllVerionsAsync

Reply:
added

Reply:
Also missing ImportKey, ImportKeyAsync

---
In reply to: [287178615](https://github.com/Azure/azure-sdk-for-net/pull/6374#discussion_r287178615) [](ancestors = 287178615)